tools/autograd/templates/VariableType.cpp

```diff
@@ -400,7 +400,7 @@ struct Flatten : IterArgs<Flatten> {
   }
 };
 
-template<typename... Args> inline variable_list flatten(Args&&... args) {
+template<typename... Args> inline variable_list flatten_tensor(Args&&... args) {
```

Comment:
What does fn this do?

Reply:
It's a function used in code gen, but I need to rename it because the compiler can't resolve it otherwise (I think it's something to do with variadic template + variadic args). I'll pick a more appropriate name.